aleph0/snippets/next13/form-api/metadata.json

```diff
@@ -2,24 +2,24 @@
   "name": "Form API",
   "description": "This snippet adds a form with the Next.js API route for submission.",
   "path": "form-api",
-  "snippets": [
+  "files": [
     {
       "name": "FORM",
       "file": "form.tsx",
       "explanation": "The form should be placed on the frontend in a `forms/<FORM>.ts` file.",
-      "knowledgeFiles": ["page.tsx"]
+      "generalSnippetReference": ["page.tsx"]
```

Comment:
not a fan of this new naming either. how about `references`?

Reply:
Yeah I didn't like it either.